eos-enable-zram.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=swap with zram
+DefaultDependencies=no
```

Comment:
Since you have `DefaultDependencies=no`, I think you'll need to add a couple more dependencies to ensure systemd does the right thing. For instance, you need to run after `ostree-remount.service` so that `modprobe zram` can find the actual module.

Actually, you may not need `DefaultDependencies=no`. Our other swap and mount units don't have that. If you leave it out, then you don't have to worry about any of this.